pyproject.toml

```
@@ -0,0 +1,140 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "example"
+description = "Short one-sentence description of the package"
+license = {file = "LICENSE"}
+readme= "README.md"
+keywords = [
+    "rubin",
+    "lsst",
+]
+# https://pypi.org/classifiers/
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Typing :: Typed",
+]
+requires-python = ">=3.11"
+dependencies = []
```

It looks like you need to bring over the dependencies originally in `setup.py`:

```suggestion
dependencies = [
    "jira",
    "tabulate",
    "flask",
    "graphviz"
]
```

---

oh snap that makes sense! I was trying to do that in the wrong way entirely lmao

---

The GitHub preview seems to be indicating I got the toml syntax wrong; not sure?

---

Yeah idk why it's highlighted red, but it looks fine to me. I looked up the toml docs and cross referenced another repo and it looks/ runs well.